frontend/src/components/organizations/NotForProfitQuestions.vue

```diff
@@ -0,0 +1,216 @@
+<template>
+  <v-skeleton-loader :loading="loading" type="table-tbody">
+    <v-card class="my-4 pa-4 w-100" variant="outlined">
+      <v-container fluid class="pa-0">
+        <AppLabel>Not for Profit Information</AppLabel>
+        <v-card variant="outlined" class="card-outline px-2 py-6 my-4">
+          <v-row no-gutters>
```

Comment:
if the label and AppDateInput are on separate lines, I think we don't need to have v-row and v-col?

Reply:
done